teambit.bvm/list/list.ts

```diff
@@ -1,24 +1,23 @@
 import os from 'os';
-import { GcpList } from './gcp';
+import { GcpList, ReleaseType } from './gcp';
 import { Config } from '@teambit/bvm.config';
 import semver from 'semver';
 import fs from 'fs-extra';
 import { LocalVersionList, RemoteVersionList } from './version-list';
 import { LocalVersion } from './version';
 
-// export enum ReleaseType {
-//   STABLE = 'stable',
-// }
+export { ReleaseType };
 
 export type ListOptions = {
   limit?: number;
-  // 'release-type'?: ReleaseType;
+  releaseType?: ReleaseType;
 };
 
 const config = Config.load();
 
 export async function listRemote(options?: ListOptions): Promise<RemoteVersionList> {
-  const gcpList = GcpList.create('dev', os.type(), process.arch, {
+  const releaseType = options?.releaseType ?? config.get('release-type');
```

Comment:
also, let's make it capital and with `_` to be consistent with other configs
like RELEASE_TYPE
We can make in the config, support for both capital or lower case

Reply:
I used release type because it was already in the code. But isn't release channel a better name for this?